Source/WebKit/Shared/WebBackForwardListItem.h

```diff
@@ -89,6 +89,9 @@ class WebBackForwardListItem : public API::ObjectImpl<API::Object::Type::BackFor
     WebBackForwardCacheEntry* backForwardCacheEntry() const { return m_backForwardCacheEntry.get(); }
     SuspendedPageProxy* suspendedPage() const;
 
+    void setIsRootChildFrameItem(bool rootChildFrameItem) { m_isRootChildFrameItem = rootChildFrameItem; }
```

Comment:
This seems like a temporary hack to me, but we'll see.  It certainly won't affect site-isolation-off behavior in the meantime.